### DISCUSSION TITLE
Source error and video stops in the middle

### DISCUSSION THREAD

**Discussion by Figim** [libre-tube/LibreTube#6898] [General]
### Steps to reproduce

**
Play video: https://youtu.be/Ze0-2_GmzWk. A video you've never played
Watched the video for about 1 minute. Then a source error occurred.**

1. SOURCE ERROR IN VIDEO IF CODEC FAILS
2. The video got stuck at that second and the player froze.

### Expected behavior

There should be no source error and the video and player should not stop. 

### Actual behavior

https://github.com/user-attachments/assets/e04270e8-c8dd-4158-a4a7-b6b415dd892b

Or

https://github.com/user-attachments/assets/96a7c739-f585-44b0-b130-b3b4872b2035

### LibreTube version

0.26.0 Debug

### Android version

All

### Other details

![Screenshot_20241219-214032_LibreTube Debug](https://github.com/user-attachments/assets/c437270a-0a03-43c8-b882-cd40753c98ac)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Figim** [comment]
Even if "All" is selected as the codec, the video sometimes stops and the player freezes. A source error occurs. What could be the reason for this?

https://youtu.be/Ze0-2_GmzWk

Piped servers are a problem. But how?

**Comment by Figim** [comment]
https://github.com/libre-tube/LibreTube/issues/3353
This seems to be the issue. However, I have the piped proxy disabled. The source error with video freeze has been bugging me for a few days. It happens after 1 minute.

**Comment by Bnyro** [comment]
This looks like a new way from YouTube to stop third party clients. You should also enable `Local streams extraction`, but I guess you probably do already.

I can't however reproduce that personally.

**Reply by Figim** [reply]
> This looks like a new way from YouTube to stop third party clients. You should also enable `Local streams extraction`, but I guess you probably do already.
> 
> I can't however reproduce that personally.

This is a newpipe problem. Maybe if they fix it, the extractor will need to be updated. But the problem only occurs for LibreTube in DASH

Same problem: https://github.com/TeamNewPipe/NewPipe/issues/11803
